### PR TITLE
Map Atlas code, much happier this time around

### PR DESCRIPTION
**Pull request by OroArmor**

### CODE REVIEW

File: mappings/com/mojang/blaze3d/texture/TextureUtil.mapping [hunk 1]
```diff
@@ -5,6 +5,9 @@ CLASS com/mojang/blaze3d/platform/TextureUtil com/mojang/blaze3d/texture/Texture
 	METHOD bind bind (I)V
 		ARG 0 id
 	METHOD generateTextureId generateTextureId ()I
+	METHOD getDebugTexturePath getDebugTexturePath ()Ljava/nio/file/Path;
```
Comment (supersaiyansubtlety) [inline mappings/com/mojang/blaze3d/texture/TextureUtil.mapping:8]:
```suggestion
	METHOD getDebugTexturePath ()Ljava/nio/file/Path;
```
I'm unclear on this but I thought the fact that the left name is readable meant it didn't need mapping.

File: mappings/com/mojang/blaze3d/texture/TextureUtil.mapping [hunk 2]
```diff
@@ -5,6 +5,9 @@ CLASS com/mojang/blaze3d/platform/TextureUtil com/mojang/blaze3d/texture/Texture
...
+	METHOD getDebugTexturePath getDebugTexturePath (Ljava/nio/file/Path;)Ljava/nio/file/Path;
```
Comment (supersaiyansubtlety) [inline mappings/com/mojang/blaze3d/texture/TextureUtil.mapping:9]:
```suggestion
	METHOD getDebugTexturePath (Ljava/nio/file/Path;)Ljava/nio/file/Path;
```

File: mappings/com/mojang/blaze3d/texture/TextureUtil.mapping [hunk 3]
```diff
@@ -33,3 +36,10 @@ CLASS com/mojang/blaze3d/platform/TextureUtil com/mojang/blaze3d/texture/Texture
 		ARG 0 inputStream
 	METHOD releaseTextureId releaseTextureId (I)V
 		ARG 0 id
+	METHOD writeAsPNG writeAsPNG (Ljava/nio/file/Path;Ljava/lang/String;IIII)V
```
Comment (supersaiyansubtlety) [inline mappings/com/mojang/blaze3d/texture/TextureUtil.mapping:39]:
```suggestion
	METHOD writeAsPNG (Ljava/nio/file/Path;Ljava/lang/String;IIII)V
```

File: mappings/net/minecraft/client/texture/SpriteAtlasManager.mapping [hunk 4]
```diff
@@ -0,0 +1,30 @@
+CLASS net/minecraft/unmapped/C_pzkdqusv net/minecraft/client/texture/SpriteAtlasManager
+	FIELD f_lodelmir atlases Ljava/util/Map;
+	METHOD <init> (Ljava/util/Map;Lnet/minecraft/unmapped/C_rglkduer;)V
+		ARG 1 atlasIds
+		ARG 2 textureManager
+	METHOD close close ()V
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/client/texture/SpriteAtlasManager.mapping:6]:
```suggestion
	METHOD close ()V
```

File: mappings/net/minecraft/client/texture/SpriteAtlasManager.mapping [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+		METHOD <init> (Lnet/minecraft/unmapped/C_qwirmgcf;Lnet/minecraft/unmapped/C_vkaguudk$C_oimhlxej;)V
+			ARG 1 atlas
+			ARG 2 preparations
+		METHOD m_hkgikvrq get (Lnet/minecraft/unmapped/C_ncpywfca;)Lnet/minecraft/unmapped/C_uflrwbwt;
+			ARG 1 id
+		METHOD m_mjyrhafx upload ()V
+		METHOD m_ujghzpzi getMissing ()Lnet/minecraft/unmapped/C_uflrwbwt;
+		METHOD m_ywtgbipa readyForUpload ()Ljava/util/concurrent/CompletableFuture;
+	CLASS C_nkqpxxfs AtlasEntry
+		METHOD close close ()V
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/client/texture/SpriteAtlasManager.mapping:30]:
```suggestion
		METHOD close ()V
```

File: mappings/net/minecraft/client/texture/SpriteContents.mapping [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+	FIELD f_izzfdnbx animatedTexture Lnet/minecraft/unmapped/C_yxnjfniw$C_jdjmjndn;
+	FIELD f_paosgjng originalImage Lnet/minecraft/unmapped/C_ayikuhxa;
+	FIELD f_wunuolws id Lnet/minecraft/unmapped/C_ncpywfca;
+	FIELD f_ziyojtxg mipmapLevels [Lnet/minecraft/unmapped/C_ayikuhxa;
+	METHOD <init> (Lnet/minecraft/unmapped/C_ncpywfca;Lnet/minecraft/unmapped/C_nrllqdla;Lnet/minecraft/unmapped/C_ayikuhxa;Lnet/minecraft/unmapped/C_upzqjtby;)V
+		ARG 1 id
+		ARG 2 size
+		ARG 3 image
+		ARG 4 animationMetadata
+	METHOD close close ()V
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/client/texture/SpriteContents.mapping:13]:
```suggestion
	METHOD close ()V
```

File: mappings/net/minecraft/client/texture/SpriteContents.mapping [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+		FIELD f_sqhnnzti subFrame I
+		FIELD f_tgpqdsxm texture Lnet/minecraft/unmapped/C_yxnjfniw$C_jdjmjndn;
+		FIELD f_xolnzcgu interpolationData Lnet/minecraft/unmapped/C_yxnjfniw$C_thgmdldj;
+		FIELD f_zprnyxqt frame I
+		METHOD <init> (Lnet/minecraft/unmapped/C_yxnjfniw;Lnet/minecraft/unmapped/C_yxnjfniw$C_jdjmjndn;Lnet/minecraft/unmapped/C_yxnjfniw$C_thgmdldj;)V
+			ARG 2 texture
+			ARG 3 interpolationData
+	CLASS C_thgmdldj InterpolationData
+		FIELD f_gddrvoox activeFrame [Lnet/minecraft/unmapped/C_ayikuhxa;
+		METHOD close close ()V
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/client/texture/SpriteContents.mapping:82]:
```suggestion
		METHOD close ()V
```

File: mappings/net/minecraft/client/texture/SpriteLoader.mapping [hunk 8]
```diff
@@ -0,0 +1,44 @@
+CLASS net/minecraft/unmapped/C_vkaguudk net/minecraft/client/texture/SpriteLoader
+	FIELD f_bsupqnaj maxSupportedTextureSize I
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/client/texture/SpriteLoader.mapping:2]:
```suggestion
	FIELD f_bsupqnaj maxSupportedTexture{Units} I
```
Would it make sense to use whatever the units are?

File: mappings/net/minecraft/client/texture/atlas/source/UnstitchSpriteSource.mapping [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+		METHOD <init> (Lnet/minecraft/unmapped/C_ncpywfca;Lnet/minecraft/unmapped/C_tuynegco;I)V
+			ARG 1 id
+			ARG 2 resource
+			ARG 3 regionCount
+		METHOD m_izmxfzcg release ()V
+		METHOD m_kpiuvtsr get ()Lnet/minecraft/unmapped/C_ayikuhxa;
+	CLASS C_arxmarph RegionGenerator
+		FIELD f_bhzsadxn image Lnet/minecraft/unmapped/C_kafxtemf$C_akjjowfy;
+		FIELD f_bibpunia region Lnet/minecraft/unmapped/C_kafxtemf$C_wpyfoogz;
+		FIELD f_gpnbyfhy yDivisor D
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/client/texture/atlas/source/UnstitchSpriteSource.mapping]: Why are these `x/yDivisor` but lines 7/8 are `divisorX/Y`?

File: mappings/com/mojang/blaze3d/texture/TextureUtil.mapping (hunk 1 above)
Comment (OroArmor) [reply mappings/com/mojang/blaze3d/texture/TextureUtil.mapping:8]: For all of these it makes it showed as mapped in enigma.

File: mappings/net/minecraft/client/texture/atlas/source/UnstitchSpriteSource.mapping [hunk 10]
```diff
@@ -0,0 +1,34 @@
...
+		FIELD f_pcbfjpdd xDivisor D
+		METHOD <init> (Lnet/minecraft/unmapped/C_kafxtemf$C_akjjowfy;Lnet/minecraft/unmapped/C_kafxtemf$C_wpyfoogz;DD)V
+			ARG 1 image
+			ARG 2 region
+			ARG 3 xDivisor
+			ARG 5 yDivisor
```
Comment (OroArmor) [inline mappings/net/minecraft/client/texture/atlas/source/UnstitchSpriteSource.mapping]:
```suggestion
		FIELD f_gpnbyfhy divisorY D
		FIELD f_pcbfjpdd divisorX D
		METHOD <init> (Lnet/minecraft/unmapped/C_kafxtemf$C_akjjowfy;Lnet/minecraft/unmapped/C_kafxtemf$C_wpyfoogz;DD)V
			ARG 1 image
			ARG 2 region
			ARG 3 divisorX
			ARG 5 divisorY
```

File: mappings/net/minecraft/client/texture/atlas/source/UnstitchSpriteSource.mapping (hunk 9 above)
Comment (OroArmor) [reply mappings/net/minecraft/client/texture/atlas/source/UnstitchSpriteSource.mapping]: ah this is an oversight, thanks, i thought something was different when mapping

**Review by supersaiyansubtlety — APPROVED** [review]

File: mappings/com/mojang/blaze3d/texture/TextureUtil.mapping (hunk 1 above)
Comment (eristhea) [reply mappings/com/mojang/blaze3d/texture/TextureUtil.mapping:8]: hm, I remember for a few of my pull requests we cleared out names like this

**Review by eristhea — APPROVED** [review]
coverage!

**Pull request merged**